packages/yargs/README.md

```diff
@@ -29,11 +29,17 @@ Options:
   --help, -h  Show help                                               [boolean]
 
 $ un start
-foo info
-server listening at http://localhost:8080
+foo listening at http://localhost:8080
```

Comment:
untool does not log such information anymore, does it? :D

Reply:
`untool` does since recently, but `@untool/yargs` still does not.